requirements.txt

```diff
@@ -8,7 +8,7 @@ PyYAML>=3.12 # MIT
 SQLAlchemy>=1.2.7 # MIT
 PyMySQL>=0.8.0 # MIT
 flask-sqlalchemy>=2.3.2 # BSD
-kubernetes>=v8.0.0 # Apache-2.0
+kubernetes>=v8.0.0,<10.0.0 # Apache-2.0
```

Comment:
this change is from this issue: https://github.com/kubernetes-client/python/issues/866